### ISSUE TITLE
Generated avatar sharing links for NEVER show profile with new front page

### ISSUE THREAD

**Issue by frillsdev** [HabitRPG/habitica#4997]
When you try to share your Habit profile using AddThis, the generated URL does not take you to your profile overlaying the front page of Habit like it used to.

It is a fairly minor issue, but I just wanted to draw your attention to it, so that you're aware.
Here is an example (my profile link generated to be sent to twitter)
https://habitrpg.com/static/front/#?memberId=2ebecf01-1cc2-4cc2-b7d1-739bbdc0e78e

I have checked it in both Chrome and Mozilla, same issue in both.

**Comment by gisikw** [comment]
Looks like https://habitrpg.com/static/front/modals/member.html is 404'ing. I'm guessing `website/views/shared/` isn't getting symlinked somewhere in the deploy process.

**Comment by lemoness** [comment]
Ping @lefnire @sabrecat! Additional issues with avatar sharing.

**Comment by hairlessbear** [comment]
This is a duplicate of #4297, right?

**Comment by Alys** [comment]
@hairlessbear Actually, no, although thank you for asking about that - we are always grateful for help with removing duplicate issues! #4297 occurs sometimes for some users. This new bug occurs always for all users and started when the home page was changed. So, when we've fixed this one, #4297 will probably remain. :-/  I'm about to edit this title to make that clearer.

@gisikw Thanks for your comment! We should look into that....
